bibliopixel/cli/version.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""
+Prints version information
+"""
+
+from bibliopixel import __name__, __version__, __status__, __copyright__
```

Comment:
More new `__` names, but only `__name__` and `__version__` appear to be actual `__` names...

Also, I have serious misgivings about importing `__main__` from elsewhere on top of `__main__` in this module.  Perhaps if I understood better what was going on...?